data/json/items/magazines.json

```diff
@@ -1,5 +1,21 @@
 [
     {   "type" : "MAGAZINE",
+        "id" : "garandclip",
+        "price" : 600,
+        "name" : "M1 Garand Clip",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A light-weight en-bloc clip for use with the M1 garand that will be ejected with a distinctive ping sound after the last round is fired.",
```

Comment:
Ok. I don't really want to otherwise hold up this PR ammo belts depend upon `MAG_DESTROY` to correctly function